TEI.js

```diff
@@ -109,6 +109,27 @@ var exportedXMLIds = {};
 var generatedItems = {};
 var allItems = {};
 
+// replace formatting with TEI tags
+function replaceFormatting (title){
+    var titleText = title;
+    // italics
+    titleText = titleText.replace(/<i>/g, '<hi rend="italics">');
+    titleText = titleText.replace(/<\/i>/g, '</hi>');
+    // bold
+    titleText = titleText.replace(/<b>/g, '<hi rend="bold">');
+    titleText = titleText.replace(/<\/b>/g, '</hi>');
+    // subscript
+    titleText = titleText.replace(/<sub>/g, '<hi rend="sub">');
+    titleText = titleText.replace(/<\/sub>/g, '</hi>');
+    // superscript
+    titleText = titleText.replace(/<sup>/g, '<hi rend="sup">');
+    titleText = titleText.replace(/<\/sup>/g, '</hi>');
+    // small caps
+    titleText = titleText.replace(/<span style="font-variant:small-caps;">|<span style="font-variant: small-caps;">/g, '<hi rend="smallcaps">');
+    titleText = titleText.replace(/<\/span>/g, '</hi>');
```

Comment:
Two suggestions here:

1. CSL also has a (deprecated?) `<sc>` element that can also be used for small caps.
2. Replacing every `</span>` with `</hi>` is going to cause problems, since e.g. it can also be used for `<span class="nocase">`.